VTEX - Marketplace APIs.json

```diff
@@ -3452,35 +3469,42 @@
                 },
                 "SellerId": {
                     "type": "integer",
-                    "description": "ID of the seller in the marketplace. This ID must be created by the marketplace and informed to the seller before the integration is built."
+                    "description": "ID of the seller in the marketplace. This ID must be created by the marketplace and informed to the seller before the integration is built.",
+                    "default": 1
                 },
                 "Height": {
                     "type": "integer",
                     "format": "decimal",
-                    "description": "Height of the SKU."
+                    "description": "Height of the SKU.",
+                    "default": 10
                 },
                 "Width": {
                     "type": "integer",
                     "format": "decimal",
-                    "description": "Width of the SKU."
+                    "description": "Width of the SKU.",
+                    "default": 10
                 },
                 "Length": {
                     "type": "integer",
                     "format": "decimal",
-                    "description": "Length of the SKU."
+                    "description": "Length of the SKU.",
+                    "default": 10
                 },
                 "WeightKg": {
                     "type": "integer",
                     "format": "decimal",
-                    "description": "Weight of the SKU in kilograms."
+                    "description": "Weight of the SKU in kilograms.",
+                    "default": 10
                 },
                 "RefId": {
                     "type": "string",
-                    "description": "SKU reference code. Mandotory if the EAN is not informed."
+                    "description": "SKU reference code. Mandotory if the EAN is not informed.",
+                    "default": 10
```

Comment:
Se isso for uma `string`, deve ser "10", se for um int, mudar o `type`

Reply:
feito!